src/interp/InterpreterUtils.ml

```diff
@@ -388,7 +388,7 @@ let compute_ids () =
         loan_ids := BorrowId.Set.add id !loan_ids
 
       method! visit_abstraction_id _ id = aids := AbstractionId.Set.add id !aids
-      method! visit_region_id _ id = rids := RegionId.Set.add id !rids
+      method! visit_free_region _ id = rids := RegionId.Set.add id !rids
```

Comment:
We should keep the `visit_region_id` (especially as some ids may not appear inside the `Free` variant but, for instance, inside a set in a region abstraction - that's actually a good reason for having a clear separation between free variables and bound variables).

Reply:
oh yeah, I wrote this expecting we'd merge the ids but that's no longer needed

Reply:
Fixed, the drawback is that it's called `visit_free_region_id` because that's what charon calls it, whereas aeneas renames it to `region_id`. I could rename to `free_region_id` in aeneas too.

Reply:
Thanks! Is there a way of renaming it to `visit_region_id`?
The reason I would like to keep it that way is that in Aeneas regions are mostly used during the symbolic execution (whenever we call a function with lifetimes, we introduce fresh regions). In that context it is a bit weird to call them "free regions"...

Reply:
I "renamed" it in the last commit.